### PR TITLE
add Person view, test components

### PR DESCRIPTION
**Pull request by Rebornix-zero**
Add PersonView, AccountView, SettingView. 
Test NavBar, PostCard, TopicCard.

### CODE REVIEW

File: src/components/NavBar/__test__/NavBar.test.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    fireEvent.press(screen.getByText("首页"));
+    await waitFor(() => {
+      expect(mockedNavigate).toBeCalledTimes(2);
+    });
+  });
+  test("NavBar Navigate test with publish", async () => {
+    render(<NavBar />);
+    fireEvent.press(screen.getByTestId("publishIcon"));
+    await waitFor(() => {
+      expect(mockpush).toBeCalledTimes(1);
```
Comment (rennsax) [inline src/components/NavBar/__test__/NavBar.test.tsx]: `mockedNavigate` is a synchronized function therefore no need to `waitFor`.

File: src/components/PostCard/__test__/PostCard.test.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import {
+  fireEvent,
+  render,
+  screen,
+  waitFor,
+} from "@testing-library/react-native";
+import PostCard from "../PostCard";
+import { Text } from "react-native";
+import * as PostCardHandler from "../../../services/postCardhandler";
```
Comment (Eitheages) [inline src/components/PostCard/__test__/PostCard.test.tsx:9]: Don't test it until you come up with a good API name/design.

File: src/components/PostCard/__test__/PostCard.test.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+const goodHandlerMock = jest.spyOn(PostCardHandler, "goodHandler");
+const badHandlerMock = jest.spyOn(PostCardHandler, "badHandler");
+const commentHandlerMock = jest.spyOn(PostCardHandler, "commentHandler");
```
Comment (rennsax) [inline src/components/PostCard/__test__/PostCard.test.tsx]: Don't test it until you come up with a good API name/design.

File: src/utils/__test__/numericCarry.test.tsx
Comment (rennsax) [inline src/utils/__test__/numericCarry.test.tsx:1]: Prefer named import, which obviously provides a cleaner code here.

**Review by rennsax — APPROVED** [review]
@WHC198025 
1. Use correct type of change (feat/test/build/...)
2. Pay attention to the file that added to index. Are they necessary/correct?
3. Spelling 😉
4. Do exactly ONE thing in your commit. And the work should be completed.

**Pull request merged**